.github/workflows/check-supported-versions.yaml

```diff
@@ -15,10 +15,6 @@ jobs:
         include:
           - java: 8
             os: windows-latest
-          - java: 17
-            os: ubuntu-latest
-            # Need to update to Gradle version with v17 support in modules/openapi-generator-gradle-plugin/pom.xml
-            flags: -am -pl modules/openapi-generator-cli -Dmaven.javadoc.skip=true -Dmaven.test.skip=true
```

Comment:
This is no longer necessary now that the project (and the Gradle plugin by extension) builds with Gradle 7.6.